.github/workflows/git-flow.yml

```
@@ -0,0 +1,179 @@
+    name: Deployment Check
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
+    if: |
+      ! contains(github.event.pull_request.body, '[x] Skip CD')
+    steps:
+      - name: Announce Deployment
+        run: echo "This will be deployed"
+
+  check-release:
+    name: Check Release
+    timeout-minutes: 5
```

actionlint (1.7.3)

when a reusable workflow is called with "uses", "timeout-minutes" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "check-release"

(syntax-check)